sqlalchemy_bigquery/base.py

```diff
@@ -654,26 +666,98 @@ def get_column_specification(self, column, **kwargs):
         return colspec
 
     def post_create_table(self, table):
+        """
+        Constructs additional SQL clauses for table creation in BigQuery.
+
+        This function processes the BigQuery dialect-specific options and generates SQL clauses for partitioning,
+        clustering, and other table options.
+
+        Args:
+            table (Table): The SQLAlchemy Table object for which the SQL is being generated.
+
+        Returns:
+            str: A string composed of SQL clauses for time partitioning, clustering, and other BigQuery specific
+                options, each separated by a newline. Returns an empty string if no such options are specified.
+
+        Raises:
+            TypeError: If the time_partitioning option is not a `TimePartitioning` object or if the clustering_fields option is not a list.
+            NoSuchColumnError: If any field specified in clustering_fields does not exist in the table.
+        """
+
         bq_opts = table.dialect_options["bigquery"]
-        opts = []
 
-        if ("description" in bq_opts) or table.comment:
-            description = process_string_literal(
-                bq_opts.get("description", table.comment)
+        options = {}
+        clauses = []
+
+        if (
+            bq_opts.get("time_partitioning") is not None
+            and bq_opts.get("range_partitioning") is not None
+        ):
+            raise ValueError(
+                "biquery_time_partitioning and bigquery_range_partitioning"
+                " dialect options are mutually exclusive."
+            )
+
+        if (time_partitioning := bq_opts.get("time_partitioning")) is not None:
+            self._raise_for_type(
+                "time_partitioning",
+                time_partitioning,
+                TimePartitioning,
             )
-            opts.append(f"description={description}")
 
-        if "friendly_name" in bq_opts:
-            opts.append(
-                "friendly_name={}".format(
-                    process_string_literal(bq_opts["friendly_name"])
+            if time_partitioning.expiration_ms:
+                options["partition_expiration_days"] = (
+                    time_partitioning.expiration_ms / 86400000
```

Comment:
```suggestion
                    time_partitioning.expiration_ms / (1000 * 60 * 60 * 24)  # 24 hours 
```

Reply:
Agree, this is what I did in the main README.